mdbenchmark/utils.py

```diff
@@ -125,3 +127,17 @@ def guess_ncores():
                        click.style('WARNING', fg='yellow', bold=True)))
         total_cores = 1
     return total_cores
+
+
+
+def detect_md_engine(modulename):
+    """Detects the MD engine based on the available modules.
+        Any newly implemeted mdengines must be added here.
+        Returns the python module.
+    """
+    if 'gromacs' in modulename:
+        return gromacs
+    elif 'namd' in modulename:
+        return namd
+    else:
+        raise RuntimeError("No Module Detected! did you specify the module?")
```

Comment:
Please write in the error message that the mdengine couldn't be detected from the module and what mdengines are known to mdbenchmark.